org.flatpak.popout3d.yml

```diff
@@ -0,0 +1,204 @@
+    builddir: true
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/project/panotools/libpano13/libpano13-2.9.19/libpano13-2.9.19.tar.gz
+        sha256: 037357383978341dea8f572a5d2a0876c5ab0a83dffda431bd393357e91d95a8
+    cleanup:
+      - /bin
+      - /share
+
+  - name: ilmbase
+    config-opts:
+      - --disable-static
+    sources:
+      - type: archive
+        url: https://download.savannah.nongnu.org/releases/openexr/ilmbase-2.2.1.tar.gz
+        sha256: cac206e63be68136ef556c2b555df659f45098c159ce24804e9d5e9e0286609e
+
+  - name: openexr
+    config-opts:
+      - --disable-static
+    sources:
+      - type: archive
+        url: https://download.savannah.nongnu.org/releases/openexr/openexr-2.2.1.tar.gz
+        sha256: 8f9a5af6131583404261931d9a5c83de0a425cb4b8b25ddab2b169fbf113aecd
+    cleanup:
+      - /bin
+      - /share/doc
+
+  - name: vigra
+    buildsystem: cmake-ninja
+    config-opts:
+       - -DWITH_OPENEXR=ON
+    sources:
+      - type: archive
+        url: https://github.com/ukoethe/vigra/archive/Version-1-11-1.tar.gz
+        sha256: b2718250d28baf1932fcbe8e30f7e4d146e751ad0e726e375a72a0cdb4e3250e
+    cleanup:
+      - /bin
+      - /doc
+      - /lib/vigra/
+      - /share
+
+  - name: exiv2
+    buildsystem: cmake
+    sources:
+      - type: git
+        url: https://github.com/Exiv2/exiv2.git
+        commit: e7ffd83af29187190da7f98dcbca8a4d70e19582
+    cleanup:
+      - /bin
+      - /share/man
+    
+  - name: boost
+    buildsystem: simple
+    build-commands:
+      - "./bootstrap.sh --prefix=/app --with-libraries=atomic,chrono,date_time,filesystem,iostreams,program_options,system,thread"
+      - "./b2 -j $FLATPAK_BUILDER_N_JOBS install"
+    sources:
+      - type: archive
+        url: https://dl.bintray.com/boostorg/release/1.68.0/source/boost_1_68_0.tar.bz2
+        sha256: 7f6130bc3cf65f56a618888ce9d5ea704fa10b462be126ad053e80e553d6d8b7
+    
+  - name: align_image_stack
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/project/hugin/hugin/hugin-2018.0/hugin-2018.0.0.tar.bz2
+        sha256: d3af0d066ac50e3bb243a175a64ecda136d87178419457e8822e11bcf0e565cb
+    cleanup:
+      - /bin/autooptimiser
+      - /bin/calibrate_lens_gui
+      - /bin/celeste_standalone
+      - /bin/checkpto
+      - /bin/cpclean
+      - /bin/cpfind
+      - /bin/deghosting_mask
+      - /bin/fulla
+      - /bin/geocpset
+      - /bin/hugin*
+      - /bin/icpfind
+      - /bin/linefind
+      - /bin/nona
+      - /bin/pano*
+      - /bin/PT*
+      - /bin/pto_*
+      - /bin/tca_correct
+      - /bin/verdandi
+      - /bin/vig_optimize
+      - /lib/libceleste*
+      - /lib/libhuginbaswwx*
+      - /lib/libicpfindlib*
+      - /lib/liblocalfeatures*
+      - /share
+
+  - name: popout3d
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/popout-apps/popout3d
```

Comment:
You should checkout a tag or a specific commit

Reply:
"exalm" wrote the manifest for me, for which I'm very grateful. I've not used Git before, so I'll have to learn how to do that.